lib/Docs/RST/RSTCopier.php

```diff
@@ -56,6 +57,10 @@ public function copyRst(Project $project, ProjectVersion $version): void
             $files = $this->rstFileRepository->getSourceFiles($language->getPath());
 
             foreach ($files as $file) {
+                if (str_ends_with($file, 'sidebar.rst')) {
```

Comment:
please be aware that this might result into all files with `sidebar.rst` to be skipped. a file like `who-to-render-sidebar.rst` will also not be copied. 

That's why my initial implementation was looking so odd.

Reply:
Fixed 👍